Week1/select.js

```diff
@@ -0,0 +1,45 @@
+'use strict;';
+var mysql = require('mysql');
+const help = require('./help');
+var connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world',
+});
+
+connection.connect();
+
+const args = process.argv;
+args.splice(0, 2);
+const retrieve = argument => args.includes(argument);
+
+
+if (retrieve('1'))
```

Comment:
You can also use `switch` here :)

Reply:
thanks for your feedback ^_^ I updated my homework.